staging/src/k8s.io/csi-translation-lib/plugins/in_tree_volume.go

```diff
@@ -59,3 +59,54 @@ type InTreePlugin interface {
 	// RepairVolumeHandle generates a correct volume handle based on node ID information.
 	RepairVolumeHandle(volumeHandle, nodeID string) (string, error)
 }
+
+// convertZones converts existing zone labels to NodeAffinity.
+func convertZones(pv *v1.PersistentVolume, topologyKey string) error {
+	zones := []string{}
+	for k, v := range pv.ObjectMeta.Labels {
+		if k == v1.LabelZoneFailureDomain {
```

Comment:
We need to carefully watch https://github.com/kubernetes/kubernetes/pull/81431 which I think is changing this constant to the GA naming